### PR TITLE
Improve Sync Times

### PR DESCRIPTION
**Pull request by ejfitzgerald**
Minor bugfix to improve sync times:

* Fixed issue with block cache being cleared on each reset state
* Increased the side of the ancestor cache used when syncing
* Corrected issue with lane services not using available threads for tx verification
* Additional logging tweaks

### CODE REVIEW

**Comment by ejfitzgerald** [comment]
FYI @pbukva

File: apps/constellation/constellation.cpp [hunk 1]
```diff
@@ -120,34 +121,34 @@ std::shared_ptr<ledger::DAGInterface> GenerateDAG(bool generate, std::string con
...
-    cfg.external_identity = std::make_shared<crypto::ECDSASigner>();
-    cfg.external_port     = start_port++;
-    cfg.external_network_id =
+    auto &shard = configs[i];
+
+    shard.lane_id           = i;
+    shard.num_lanes         = cfg.num_lanes();
+    shard.storage_path      = cfg.db_prefix;
+    shard.external_identity = std::make_shared<crypto::ECDSASigner>();
+    shard.external_port     = start_port++;
```
Comment (bipll) [inline apps/constellation/constellation.cpp]: Or `start_port + i`?

File: libs/ledger/src/chain/block_coordinator.cpp [hunk 2]
```diff
@@ -1129,7 +1130,6 @@ BlockCoordinator::State BlockCoordinator::OnReset()
   current_block_.reset();
   next_block_.reset();
   pending_txs_.reset();
-  blocks_to_common_ancestor_.clear();
```
Comment (n-hutton) [inline libs/ledger/src/chain/block_coordinator.cpp:1132]: 👍

File: libs/ledger/src/storage_unit/storage_unit_client.cpp [hunk 3]
```diff
@@ -183,7 +183,7 @@ bool StorageUnitClient::RevertToHash(Hash const &hash, uint64_t index)
   {
     assert(!hash.empty());
 
-    FETCH_LOG_INFO(LOGGING_NAME, "reverting tree leaf: ", lane_merkle_hash.ToHex());
+    FETCH_LOG_DEBUG(LOGGING_NAME, "reverting tree leaf: ", lane_merkle_hash.ToHex());
```
Comment (n-hutton) [inline libs/ledger/src/storage_unit/storage_unit_client.cpp:186]: I would leaf this as an info as it only happens on a fork/revert right, so not that common a code path in our system

**Review by n-hutton — APPROVED** [review]

**Review by pbukva — APPROVED** [review]
LGTM

**Pull request merged**